src/checker/hunspell.rs

```diff
@@ -98,6 +99,21 @@ fn cache_builtin() -> Result<(PathBuf, PathBuf)> {
     Ok((path_dic, path_aff))
 }
 
+/// The value is `true` if string is made of emoji's
+/// or unicode `VULGAR FRACTION`.
+pub fn consists_of_vulgar_fractions_or_emojis(word: &str) -> bool {
+    lazy_static! {
+        static ref VULAGAR_OR_EMOJI: regex::RegexSet = regex::RegexSetBuilder::new(&[
```

Comment:
VULGAR_OR_EMOJI?

Reply:
Hmm, I thought I fixed that already :monocle_face: :coffee: